src/ResourceGraph/ResourceGraph/Models/PSResourceGraphResponse.cs

```diff
@@ -0,0 +1,30 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.ResourceGraph.Models
+{
+    using System.Collections.Generic;
+    using System.Management.Automation;
+    using Microsoft.WindowsAzure.Commands.Common.Attributes;
+
+    public class PSResourceGraphResponse
```

Comment:
>PSResourceGraphResponse [](http://example.com/codeflow?start=17&length=23)

now we have a response object; should we include total records and count fields as well.